src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,75 @@
-import React from 'react';
+import {
+  ChangeEvent,
+  FC,
+  FormEvent,
+  useState,
+} from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+interface Props {
+  newMovieHandler: (movie: Movie) => void;
+}
+
+export const FindMovie: FC<Props> = ({ newMovieHandler }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState<string>('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
```

Comment:
TS already knows about these types, bc you put an initial value
```suggestion
  const [query, setQuery] = useState('');
  const [isLoading, setIsLoading] = useState(false);
  const [isError, setIsError] = useState(false);
```